modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc

```diff
@@ -5,109 +5,109 @@
 
 == Introduction
 
-Federated Cloud Sharing is managed by the Federation app. 
-When you enable the Federation app you can easily and securely link file shares between ownCloud servers, in effect creating a "cloud" of ownCloud installations.
+Federated Cloud Sharing is managed by the Federation app. When you enable the Federation app you can easily and securely link file shares between ownCloud servers, in effect creating a "cloud" of ownCloud installations.
 
-[IMPORTANT]
-====
-For security reasons federated sharing **strictly requires HTTPS (SSL/TLS)**.
-====
+IMPORTANT: For security reasons federated sharing **strictly requires HTTPS (SSL/TLS)**.
 
-IMPORTANT: We strongly recommend using HTTP for development and testing purposes. 
-However, to do so, you have to set `'sharing.federation.allowHttpFallback' => true,` in **config/config.php**.
+IMPORTANT: We strongly recommend using HTTP for development and testing purposes. However, to do so, you have to set `'sharing.federation.allowHttpFallback' => true,` in **config/config.php**.
 
 == Configuration
 
 Follow these steps to establish a trusted connection between two servers.
 
-. Verify that both servers have SSL certificates.
-  If you open the server URL in your browser and see a lock icon on the left-hand side of the address bar, the certificate is valid.
+. Verify that both servers have SSL certificates. +
```

Comment:
Is the "+" supposed to be there?

Reply:
Yes. It tells the (numbered ) list item that the current sentence ends here, but the next line(s) are part of the item and therefore properly intented. (soft break). I can make you a build on staging so you can "optical review" this. Before commiting this change, a final local build rendered fine.